lua/nvim-tundra/hl_group/plugins/nvimtree.lua

```diff
@@ -0,0 +1,10 @@
+local M = {}
+
+M.map = function(ss, opts)
+  return {
+    NvimTreeNormal = { fg = ss.filetree.fg, bg = ss.filetree.bg },
+    NvimTreeVertSplit = ss.filetree.border,
```

Comment:
Change to use base stylesheet groups instead of a new `filetree` group. I've also changed the background colour of the `nvim-tree` window so that there is a less drastic contrast between it and normal windows.

Before: 
<img width="2132" alt="image" src="https://user-images.githubusercontent.com/43378933/192095996-ded839bf-447d-425a-89e8-5e5d147fbfdb.png">

After:
<img width="2132" alt="image" src="https://user-images.githubusercontent.com/43378933/192096056-b5588579-b6f3-4aa7-8ed1-1938f8219582.png">


```suggestion
    NvimTreeNormal = { fg = ss.fg.normal, bg = ss.bg.dimmed },
    NvimTreeVertSplit = { fg = ss.bg.dimmed, bg = ss.bg.dimmed },
```